api.proto

```diff
@@ -1,6 +1,9 @@
 syntax = "proto3";
 
+import "google/protobuf/timestamp.proto";
+
 package api;
+option go_package = "/api";
```

Comment:
Does setting the `go_package` help us use the generated code in goval?

Reply:
Oops no, i just forgot to remove it.